README.md

```diff
@@ -314,6 +314,20 @@ Input parameter - name (string) specifying service name, pod name, replication c
 client.delete_service("redis-service")
 ```
 
+If you want to cascade delete, for example a deployment, you can use the `delete_options` parameter.
+
+```ruby
+deployment_name = 'redis-deployment'
+namespace = 'default'
+delete_options = Kubeclient::Resource.new(
+    apiVersion: 'meta/v1',
+    gracePeriodSeconds: 0,
+    kind: 'DeleteOptions',
+    propagationPolicy: 'Foreground' # Orphan, Foreground, or Background
```

Comment:
can we encapsulate these values with an enum/const type? e.g. `Kubeclient::PropagationPolicy::Foreground`?

Reply:
I want to differ, simple string LGTM here.
What if k8s later adds more values?  I don't see a benefit to kubeclient maintaining this set in sync with the k8s reality (especially if reality can differ by server version).

Also, if the values these consts will have are still the strings 'Foreground' etc, we can always add such consts later, need not block this PR.

Reply:
agree @cben

Reply:
OK that's fair

Reply:
2-space indent ?